website/docs/r/virtual_machine.html.markdown

```diff
@@ -389,6 +389,16 @@ The following attributes are exported:
 
 * `id` - The ID of the Virtual Machine.
 
+* `identity` - An `identity` block as defined below, which contains the Managed Service Identity information for this Virtual Machine.
+
+---
+
+A `identity` block exports the following:
+
+* `principal_id` - The Principal ID for the Service Principal associated with the Managed Service Identity of this Virtual Machine.
+
+-> You can access the Principal ID via `${azurerm_virtual_machine.test.identity.principal_id}`
```

Comment:
since `identity` is a List this'll be:

```suggestion
-> You can access the Principal ID via `${azurerm_virtual_machine.test.identity.0.principal_id}`
```